File: Data Structures & Algorithms/max-consecutive-ones/submission-1.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
       int res = 0, cnt = 0;
        for (int num : nums) {
            if(num==1){
                cnt+=1;
            }
            else{
                cnt = 0;
            }
            res = Math.max(res, cnt);

        }
        return res;

}
}

File: Data Structures & Algorithms/maximum-subarray/submission-1.py
class Solution:
    def maxSubArray(self, nums: List[int]) -> int:
        largestSum = nums[0]
        currentSum = 0

        for num in nums:
            if currentSum < 0:
                currentSum = 0
            currentSum+=num
            largestSum = max(largestSum, currentSum)
        return largestSum